php/ui/component/class-cache-status.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Cache Status UI Component.
+ *
+ * @package Cloudinary
+ */
+
+namespace Cloudinary\UI\Component;
+
+use Cloudinary\Cache;
+use Cloudinary\Cache\Cache_Point;
+
+/**
+ * Cache Status Component to render plan status.
+ *
+ * @package Cloudinary\UI
+ */
+class Cache_Status extends media_status {
```

Comment:
@DavidCramer 

Can we update the extended class name to `Media_Status`?

Reply:
Oops! Will do.